userspace/libsinsp/CMakeLists.txt

```diff
@@ -309,8 +309,8 @@ if(NOT WIN32)
 			target_link_libraries(sinsp INTERFACE "${ONIGURUMA_LIB}")
 			list(APPEND SINSP_PKGCONFIG_LIBRARIES "${ONIGURUMA_LIB}")
 			add_dependencies(sinsp jq)
-		endif()
-	endif() # NOT APPLE
+		endif() # NOT APPLE
+	endif() # NOT WIN32
```

Comment:
It still looks to me that this is `NOT APPLE`

Reply:
Fixed. Thanks!